docs/3.2/trustedclusters.md

```diff
@@ -52,6 +52,39 @@ $ tsh clusters
 Trusted clusters also have their own restrictions on user access, i.e.
 _permissions mapping_ takes place.
 
+## Cluster Setup 
+
+When you first setup a cluster it's important to correctly set the Cluster name.
```

Comment:
probably Cluster -> cluster for consistency (it's not a proper noun anywhere else)